Makefile

```diff
@@ -102,12 +102,17 @@ vet: ## Run go vet against code.
 ## --------------------------------------
 
 .PHONY: test
-test: manifests generate fmt vet local-unit-test
+test: manifests generate fmt vet local-unit-test integration-test
 
 .PHONY: local-unit-test
 local-unit-test: $(ENVTEST) ## Run tests.
 	CGO_ENABLED=1 KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./pkg/... -race -coverprofile=coverage.xml -covermode=atomic -v
 
+.PHONY: integration-test
+integration-test: $(ENVTEST) ## Run integration tests.
```

Comment:
we cannot run all the tests under ./test/... which includes e2e tests,

Could you please refer to this https://github.com/Azure/fleet/blob/main/Makefile#L151

Reply:
Running all tests under ./test/apis/ instead now